kb/sd_of_ann_actions/concepts/concept_number_oriented_set.scs

```diff
@@ -0,0 +1,12 @@
+concept_number_oriented_set
+=> nrel_main_idtf:
+    [ориентированное множество чисел](* <- lang_ru;; *);
+    [oriented set of numbers](* <- lang_en;; *);
+=> nrel_idtf:
+    [ормножество чисел](* <- lang_ru;; *);
+    [orset of numbers](* <- lang_en;; *);
+<= nrel_inclusion:
+    number;
+    oriented_set;
```

Comment:
can't be included to both. This is orientd set. Number will be used as used constant for note to this concept